tests/components/Microsoft.DotNet.UpgradeAssistant.MSBuild.Tests/SdkCollectionTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Autofac.Extras.Moq;
+using Microsoft.Build.Construction;
+using Xunit;
+
+namespace Microsoft.DotNet.UpgradeAssistant.MSBuild.Tests
+{
+    [Collection(MSBuildStepTestCollection.Name)]
+    public class SdkCollectionTests
+    {
+        private const string DefaultSDK = "Microsoft.NET.Sdk";
+
+        [InlineData(DefaultSDK)]
+        [Theory]
+        public void SdkCollectionContainsTest(string sdk)
```

Comment:
What about tests for empty strings, for removals, etc. What's the code coverage of this look like (for the SdkCollection type)?

Reply:
There is a test for remove, I'll add an entry for empty string.

Reply:
You may want to add a test for remove that doesn't have the entry - that should return false

Reply:
Added